op-devstack/dsl/proofs/fault_dispute_game.go

```diff
@@ -59,8 +60,11 @@ func (g *FaultDisputeGame) Attack(eoa *dsl.EOA, claimIdx int64, newClaim common.
 	newPosition := claim.Position.Attack().ToGIndex()
 	requiredBond := contract.Read(g.game.GetRequiredBond((*bindings.Uint128)(newPosition)))
 
-	receipt := contract.Write(eoa, g.game.Attack(claim.Value, big.NewInt(claimIdx), newClaim), txplan.WithValue(requiredBond))
-	g.require.Equal(types.ReceiptStatusSuccessful, receipt.Status)
+	attackCall := g.game.Attack(claim.Value, big.NewInt(claimIdx), newClaim)
+
+	receipt, err := contract.MaybeWrite(eoa, attackCall, txplan.WithValue(requiredBond), txplan.WithGasRatio(2))
+	g.t.Require().NoErrorf(err, "error sending tx: %v", errutil.TryAddRevertReason(err))
+	g.t.Require().Equal(receipt.Status, types.ReceiptStatusSuccessful)
```

Comment:
We can simplify by reverting these changes and just using `contract.Write` as originally written.